src/openrct2/entity/Peep.h

```diff
@@ -335,6 +335,7 @@ struct Peep : EntityBase
             uint8_t TimeToStand;
             uint8_t StandingFlags;
         };
+        uint8_t TimesSlidDown;
```

Comment:
```suggestion
        uint8_t timesSlidDown;
```
New members should adhere to the current code style (which prescribed `lowerCamelCase`.